### PR TITLE
MET significance patch (74X)

### PR DESCRIPTION
**Pull request by mmarionncern**
Fix the last issues raised in https://github.com/cms-sw/cmssw/pull/7163, except for the dictionnary problem and the format of the output of the standalone producer.

### CODE REVIEW

File: RecoMET/METAlgorithms/src/METSignificance.cc [hunk 1]
```diff
@@ -90,17 +85,13 @@ metsig::METSignificance::getCovariance(const edm::View<reco::Jet>& jets,
       double jpt  = jet->pt();
       double jeta = jet->eta();
       double feta = std::abs(jeta);
-      double c = std::cos(jet->phi());
-      double s = std::sin(jet->phi());
+      double c = jet->px()/jet->pt();
```
Comment (slava77) [inline RecoMET/METAlgorithms/src/METSignificance.cc]: jpt is computed already, why derive it two more times?

**Comment by slava77** [comment]
Matthieu,
It sounds like the storage issue will be resolved today.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
@deguio 

jenkins wflow 4.22 here shows a different way of DT FED monitoring: this time things appear
wf422_dtfedsize

It looks like there are some random or undefined variables in the recently merged DT DQM code

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/7578	 9fcc25b

- code: most notable: fixe the TF1 churning issue
- comparisons: since relevant code is not running, there are no changes in monitored quantities based on jenkins report.
- expectations: once we get  the framework functionality to store SMatrix, there will be one more update.

File: RecoMET/METAlgorithms/src/METSignificance.cc (hunk 1 above)
Comment (slava77) [inline RecoMET/METAlgorithms/src/METSignificance.cc]: chances are that the compiler optimizes away repeated calls, but it's unclear if it does it in this particular case (several nested operator calls).

**Comment by davidlange6** [comment]
+1

File: RecoMET/METAlgorithms/src/METSignificance.cc (hunk 1 above)
Comment (mmarionncern) [inline RecoMET/METAlgorithms/src/METSignificance.cc]: I missed that one, will be corrected with the output format once https://github.com/cms-sw/cmssw/pull/7586 is merged

**Comment by deguio** [comment]
@slava77 
I noticed this as well. I will try to figure out what's wrong. the problem must come from the DTDataIntegrityTask.cc

thanks for reporting,
F.

**Pull request merged**